### PR TITLE
15-move-datasets-to-google-drive

### PR DESCRIPTION
**Pull request by SzymonKozl**

### CODE REVIEW

File: research/preprocessing/clustering/cluster_research.ipynb [hunk 1]
```diff
@@ -10,6 +10,15 @@
     "The goal of this notebook is to find and test different ways of clustering phone screen view data as a preprocessing step before further extraction and classification. The main focus is placed on spacial data."
    ]
   },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "### Data\n",
+    "this notebook requires `coupons_0` dataset downloaded and moved to `./data` folder. Please follow dataset download instructions from Way of Work doc."
```
Comment (GustawB) [inline research/preprocessing/clustering/cluster_research.ipynb:18]:
```suggestion
    "this notebook requires `coupons_0` dataset downloaded and moved to `./data` folder. Please follow dataset download instructions from the README in docs/."
```

File: docs/README.md [hunk 2]
```diff
@@ -4,5 +4,21 @@
...
+6. Datasets given to us by Murmuras should not be pushed to repo. There is separate tool for downloading them described below.
+## Datasets
+### Getting credentials
+Datasets are stored on google drive with limited access. To download them ask SzymonKozl for credentials and put them inside `tools/gdrive_credentials.json` file.
+### Dependencies install
+Script requires dependencies from `tools/requirements.txt`. Install them.
+### Using script
+For downloading dataset into `datasets/<ds_name>` directory run command
+```bash
+pyhton tools/data_load.py <ds_name>
```
Comment (GustawB) [inline docs/README.md:16]:
```suggestion
python tools/data_load.py <ds_name>
```

**Review by GustawB — CHANGES_REQUESTED** [review]
Waiting for credentials to check if it works; but rn LGTM.

File: tools/data_load.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import io
+from sys import argv
+import os
+
+from googleapiclient.discovery import build
+from googleapiclient.http import MediaIoBaseDownload
+from google.oauth2 import service_account
+
+
+CREDENTIALS_PATH = 'tools/gdrive_credentials.json'
```
Comment (GustawB) [inline tools/data_load.py:10]:
```suggestion
CREDENTIALS_PATH = 'gdrive_credentials.json'
```

gdrive_credentials will be in the same folder as this script (at least if I understood the README correctly). So, revious version doesn't work.

**Review by GustawB — CHANGES_REQUESTED** [review]

File: tools/data_load.py (hunk 3 above)
Comment (SzymonKozl) [reply tools/data_load.py]: changed

**Pull request merged**